s2spy/rgdr/rgdr.py

```diff
@@ -340,8 +343,8 @@ def __init__(
                 be considered as in the neighborhood of the other. This is not a maximum
```

Comment:
eps_km (float): The maximum distance (in km) between two samples for one to
                 be considered as in the neighborhood of the other. This is not a maximum bound on the distances of points within a cluster. This is the most  important DBSCAN parameter to choose appropriately. The appropriate value scales with resolution (as samples will be closer to each other). With a higher resolution, eps_km can be set to lower values and vice versa. Typical values are on the order of $\approx 10^2$.

Reply:
It my intention to communicate that distances_eps is generally roughly between 100 and 500. Distances that are smaller than 100 by 100 (smaller then 1 by 1 degree) will likely result in impractical results. Can we perhaps add a link to the documentation where we add a few figure on what happens at the edge cases and what is reasonable?

Reply:
I think an appropriate minimum bound for epsilon is the (maximum) width of a gridcell, i.e.:
$\varepsilon_{min} = \textrm{max}(\Delta\textrm{lon}, \Delta\textrm{lat})$

 The upper bound depends on how you want to group clusters by region. Setting it to 1000+ km can make a lot of sense if you would like to group all points within a certain region together, this would also reduce the chance of having a lot of similar small clusters.

What about:
```
eps_km (float): The maximum distance (in km) between two samples for them to be considered 
    to be in the neighborhood of the other. This is not a maximum bound on the distances of points 
    within a cluster.
    The minimum appropriate value is equal to the maximum width/height of a grid cell (i.e. the width of 
    the grid cell closest to the equator). The upper bound depends on when cells or clusters would
    still be considered in the same geographical region.
    Higher values will lead to fewer clusters, and/or clusters that are discontinuous geographically.
```

Reply:
I like where this is going: Maybe a minor change related to the word 'geographical region' because it can be confusing I think. 

```
eps_km (float): The maximum distance (in km) between two grid cells for them to be considered 
    to be in the same cluster. This is not a maximum bound on the distances between grid cells
    within a cluster.
    The minimum appropriate value is equal to the maximum width/height of a grid cell (i.e. the width of 
    the grid cell closest to the equator). The upper bound depends on when cells or clusters would
    still be considered to be part of the same climate signal (i.e., precursor region).
    Higher values can lead to fewer clusters, but also clusters in which the cells of the same cluster are separated by large geographical distances.
```